nexus_workcell_orchestrator/src/workcell_orchestrator.cpp

```diff
@@ -245,15 +245,15 @@ auto WorkcellOrchestrator::on_cleanup(
   this->_signal_wc_srv.reset();
   this->_task_doable_srv.reset();
   this->_cmd_server.reset();
-  this->_ctx_mgr.reset();
+  this->_job_mgr->halt_all_jobs();
   RCLCPP_INFO(this->get_logger(), "Cleaned up");
   return CallbackReturn::SUCCESS;
 }
 
 auto WorkcellOrchestrator::_configure(
   const rclcpp_lifecycle::State& /* previous_state */) -> CallbackReturn
 {
-  this->_ctx_mgr = std::make_shared<ContextManager>();
+  this->_job_mgr = JobManager(this->shared_from_this(), 1);
```

Comment:
We don't have a workcell to be able to execute multiple tasks at once so it is hardcoded. The option is there so that system orchestrator can reuse the code in the future.